src/main/java/com/google/sps/servlets/StartGame.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.sps.daos.RoomDao;
+
+@WebServlet("/startGame")
+public class StartGame extends HttpServlet {
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException { 
+
+    // Activate room
+    RoomDao dao = (RoomDao) this.getServletContext().getAttribute("roomDao");
+    dao.startGame();
+    //Todo: get room's game and get questions
+
+    response.getWriter().println(dao.getActualRoom());
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+  }
```

Comment:
Delete?

Reply:
I think she has it in doGet for now, but later it will be in doPost because you send a POST request to start a new game right?

Reply:
Yes exactly I think it's going to be useful later but I can delete it for now